internal/pkg/auth/storage.go

```diff
@@ -44,6 +44,23 @@ const (
 	AUTH_FLOW_SERVICE_ACCOUNT_KEY   AuthFlow     = "sa_key"
 )
 
+// Returns all auth field keys managed by the auth storage
+func getAllAuthFieldKeys() []authFieldKey {
+	return []authFieldKey{
+		SESSION_EXPIRES_AT_UNIX,
+		ACCESS_TOKEN,
+		REFRESH_TOKEN,
+		SERVICE_ACCOUNT_TOKEN,
+		SERVICE_ACCOUNT_EMAIL,
+		USER_EMAIL,
+		SERVICE_ACCOUNT_KEY,
+		PRIVATE_KEY,
+		TOKEN_CUSTOM_ENDPOINT,
+		JWKS_CUSTOM_ENDPOINT,
+		authFlowType,
+	}
+}
```

Comment:
For consistency with the array of config keys in `config.go`


```suggestion
var authFieldKeys = []authFieldKey{
		SESSION_EXPIRES_AT_UNIX,
		ACCESS_TOKEN,
		REFRESH_TOKEN,
		SERVICE_ACCOUNT_TOKEN,
		SERVICE_ACCOUNT_EMAIL,
		USER_EMAIL,
		SERVICE_ACCOUNT_KEY,
		PRIVATE_KEY,
		TOKEN_CUSTOM_ENDPOINT,
		JWKS_CUSTOM_ENDPOINT,
		authFlowType,
}
```